### PR TITLE
Simplified Chinese translation (ics/ibc/*)

### PR DESCRIPTION
**Pull request by en**
Hi @cwgoes ,

@arthaszeng, @Satoshi-Kusumoto and I plan to translate key chapters of the IBC specification into Simplified Chinese. The whole project consists of 3 phases:

1. Revise existing translations and translate the contents of https://github.com/cosmos/ics/tree/master/ibc directory.
2. Translate the contents of https://github.com/cosmos/ics/tree/master/spec directory.
3. Translate other content that needs to be translated.

This PR focuses on the first phase and is open for comments.

### CODE REVIEW

**Comment by en** [comment]
The translation is almost complete, but some sentences are not smooth. Need reviewing.

**Comment by toliuyi** [comment]
I think it's a key step to make IBC known by the China crypto community.

File: translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+**有关一组示例用例，请参见[此处](./4_IBC_USECASES.md) 。**
+
+**有关设计模式的讨论，请参见[此处](./5_IBC_DESIGN_PATTERNS.md) 。**
+
+本文档以简明的英语提供了整个IBC规范集中使用的关键术语的定义。
+
+## 抽象定义
+
+### 参与者
```
Comment (chengwenxi) [inline translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md]: How about also providing plain English for key terms? e.g. 
### Actor 参与者

Comment (en) [reply translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md]: Good suggestion!

File: translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+*有界交易活性*是指在特定范围内拥有交易活性。
+
+### 仅一次保证
+
+*仅一次保证*是指一个数据包仅确认不超过一次（通常仅一次保证意味着最终交易活性）。
+
+### 送达或超时保证
+
+*送达或超时保证*是指数据包或者被送达并执行，或者以给发件人回传响应的方式超时。
```
Comment (Funkyswing) [inline translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md]: A suggestion：

> *送达或超时保证*是指数据包发送者获得数据包已被送达并执行，或数据包已超时最终结果的特性。

Comment (en) [reply translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md]: Thanks for your review. The [latest version](https://github.com/cdot-network/ics/pull/22/files#diff-0fd33d4fad07d7901a8c918c1b2ff17fR146) is 

> *送达或超时保证*是指数据包要么被送达并执行，要么可对发送者证明超时。

Do you think it is better?

Comment (Funkyswing) [reply translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md]: If the source means:

> ... the sender gets confirmation that the packet is either delivered & executed or the packet is timeout...

I prefer to mine. 

If it means:

> ... the packet is either delivered & executed or the sender gets confirmation that the packet is timeout...

the latest version is right.

File: translation/zh-CN/ibc/2_IBC_ARCHITECTURE.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+IBC是一种端到端，面向连接的状态协议，用于在独立计算机上的模块之间进行可靠的，可选的有序身份认证的通信。预计IBC实现将与主机状态机上的更高级别的模块和协议共存。承载IBC的状态机必须提供一组特定的功能来验证共识记录和生成加密承诺证明，并且IBC数据包中继器（链下进程）应能够访问网络协议和物理数据链接，以读取一台机器的状态，然后将数据提交给另一台机器。
+
+## 范围
+
+IBC处理在独立计算机上的模块之间中继的负责身份认证，传输和排序的结构化数据包。该协议是在两台计算机上的模块之间定义的，但设计用于在以任意拓扑连接的任意数量的计算机上的任意数量的模块之间安全地同时使用。
+
+## 接口
+
+IBC位于智能合约，其他状态机组件或状态机上其他独立的应用程序逻辑部分，和基础共识协议，机器和网络基础结构（例如TCP / IP）等模块之间。
```
Comment (lesterli) [inline translation/zh-CN/ibc/2_IBC_ARCHITECTURE.md]:
```suggestion
IBC位于两类模块之间，一类是智能合约，其他状态机组件或状态机上其他独立的应用程序逻辑部分，另一类是基础共识协议，机器和网络基础结构（例如TCP/IP）。
```

File: translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md (hunk 2 above)
Comment (en) [reply translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md]: Yes, I think you are right. According to ics-004, there is indeed an explicit acknowledgement after the package has been delivered. Your changes will be included in the next commit.

File: translation/zh-CN/ibc/2_IBC_ARCHITECTURE.md (hunk 3 above)
Comment (en) [reply translation/zh-CN/ibc/2_IBC_ARCHITECTURE.md]: Thanks, accept.

**Comment by en** [comment]
I think this PR is ready to be merged.

**Comment by cwgoes** [comment]
@en 謝謝你!

**Pull request merged**